src/lib.rs

```diff
@@ -291,7 +291,7 @@ impl OpenzwaveAdapter {
                     ZWaveNotification::ValueAdded(value)           => {
                         if value.get_genre() != ValueGenre::ValueGenre_User { continue }
 
-                        let value_id = format!("OpenZWave/{} ({})", value.get_id(), value.get_label());
+                        let value_id = format!("OpenZWave-{:08x}-{:016x} ({})", value.get_home_id(), value.get_id(), value.get_label());
```

Comment:
ah right; I remembered we had everything in ValueID but I forgot it was not all in the id. Thanks !